src/main/proto/netflix/titus/titus_cluster_membership_api.proto

```diff
@@ -0,0 +1,205 @@
+syntax = "proto3";
+
+// Titus Membership Management API specification. This API provides access to Titus cluster operational state,
+// as well as provides management operations. The membership management API is intended to be used by internal tools,
+// which should make the calls directly to the desired members (not via proxy or load balancer). The API calls tunneled
+// via gateway are executed against any available cluster member. All members should have identical eventually
+// consistent state.
+//
+
+package com.netflix.titus;
+
+import "google/protobuf/empty.proto";
+import "netflix/titus/titus_base.proto";
+
+option java_multiple_files = true;
+option java_package = "com.netflix.titus.grpc.protogen";
+option java_outer_classname = "ClusterMembershipProto";
+
+option go_package = "titus";
+
+// ----------------------------------------------------------------------------
+// Titus supervisor data model.
+
+message ClusterMemberAddress {
+
+    string ipAddress = 1;
```

Comment:
rename to `host`, or `uri`, since it could be a resolvable name (e.g.: through DNS)?

Reply:
The idea is that if a node has many interfaces/IPs we point which one should be used. There is no valid use case yet, but it may be in the future.